index.html

```diff
@@ -47,132 +45,38 @@ <h1>Johnson Macharia</h1>
-      </div>
-
-      <div class="project-card project-card-1">
-        <img class="preview" src="icons/Rectangle 21.png">
-        <div class="project-infor">
-          <p class="project-name">
-            Project name goes here.
-          </p>
-          <ul class="tech-list">
-            <li>html/css</li>
-            <li>Ruby on Rails</li>
-            <li>JavaScript</li>
-          </ul>
-          <a href="#" class="see-project">See this project</a>
-        </div>
-      </div>
-
-      <div class="project-card project-card-2">
-        <img class="preview" src="icons/Rectangle 21 (1).png">
-        <div class="project-infor">
-          <p class="project-name">
-            Project name goes here.
-          </p>
-          <ul class="tech-list">
-            <li>html/css</li>
-            <li>Ruby on Rails</li>
-            <li>JavaScript</li>
-          </ul>
-          <a href="#" class="see-project">See this project</a>
-        </div>
-      </div>
 
-      <div class="project-card project-card-3">
-        <img class="preview" src="icons/Rectangle 21 (2).png">
-        <div class="project-infor">
-          <p class="project-name">
-            Project name goes here.
-          </p>
-          <ul class="tech-list">
-            <li>html/css</li>
-            <li>Ruby on Rails</li>
-            <li>JavaScript</li>
-          </ul>
-          <a href="#" class="see-project">See this project</a>
-        </div>
-      </div>
-
-      <div class="project-card project-card-4">
-        <img class="preview" src="icons/Rectangle 21 (3).png">
-        <div class="project-infor">
-          <p class="project-name">
-            Project name goes here.
-          </p>
-          <ul class="tech-list">
-            <li>html/css</li>
-            <li>Ruby on Rails</li>
-            <li>JavaScript</li>
-          </ul>
-          <a href="#" class="see-project">See this project</a>
-        </div>
-      </div>
-
-      <div class="project-card project-card-5">
-        <img class="preview" src="icons/Rectangle 21 (4).png">
-        <div class="project-infor">
-          <p class="project-name">
-            Project name goes here.
-          </p>
-          <ul class="tech-list">
-            <li>html/css</li>
-            <li>Ruby on Rails</li>
-            <li>JavaScript</li>
-          </ul>
-          <a href="#" class="see-project">See this project</a>
-        </div>
-      </div>
-
-      <div class="project-card project-card-6">
-        <img class="preview" src="icons/Rectangle 21 (5).png">
-        <div class="project-infor">
-          <p class="project-name">
-            Project name goes here.
-          </p>
-          <ul class="tech-list">
-            <li>html/css</li>
-            <li>Ruby on Rails</li>
-            <li>JavaScript</li>
-          </ul>
-          <a href="#" class="see-project">See this project</a>
-        </div>
-      </div>
+    <section id="project-container" class="">
+      <h2>Projects</h2>
```

Comment:
- [ ] Kindly match with the Figma design, the title and list of skills should be aligned to center in the mobile version
- [ ] Review the font size in this section, seems to be a bit bigger.

![image](https://github.com/JohnteFresh/Portfolio-Development/assets/93630700/5079817e-8ede-4986-91c6-fa4dc104f4dd)